src/climate_learn/models/modules/forecast.py

```diff
@@ -10,12 +10,14 @@
 from .utils.lr_scheduler import LinearWarmupCosineAnnealingLR
 from .utils.metrics import lat_weighted_acc, lat_weighted_mse, lat_weighted_rmse
 
+OptimizerCallable = Callable[[Iterable], torch.optim.Optimizer]
```

Comment:
Can you explain why we have `Iterable` here?

Reply:
I don't know exactly why. I referred it from [here](https://pytorch-lightning.readthedocs.io/en/latest/cli/lightning_cli_advanced_3.html#multiple-optimizers-and-schedulers).

Reply:
OK, sgtm then.